google_pubsub_adapter/publisher/google_pubsub_publisher.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from google.api_core.exceptions import AlreadyExists
+from google.cloud.pubsub_v1 import PublisherClient
+from django.conf import settings
+from rest_framework.renderers import JSONRenderer
+from herbie_core.services.message_publisher.abstract_publisher import AbstractPublisher
+from herbie_core.models.message_models_and_serializers import Message
+
+
+class GooglePubsubPublisher(AbstractPublisher):
+
+    def __init__(self):
+        self._logger = logging.getLogger(__name__)
+        self._publisher = PublisherClient()
+
+    def get_name(self) -> str:
+        return 'google_pubsub'
+
+    def send_message(self, message: Message):
+        serializer = message.get_serializer()
+
+        json_data = JSONRenderer().render(serializer.data)
+
+        topic_path = self._publisher.topic_path(settings.GCLOUD_PUBSUB_PROJECT_ID, serializer.data['type'])
```

Comment:
Like you suggested yesterday, probably it would be cool to have some kind of validation for the setting-values.

Reply:
I've added as a documentation that we should define this in the Django app. Marking as resolved for now.